portfolio/src/main/webapp/script.js

```diff
@@ -56,3 +56,25 @@ function getRandomName() {
     document.getElementById('name-container').innerText = name;
   });
 }
+
+/**
+ * Fetches names from DataServlet and adds them to the DOM as a list.
+ */
+function getNames() {
+  fetch('/data').then(response => response.json()).then((names) => {
+
+    const namesListElement = document.getElementById('names-container');
+    namesListElement.innerHTML = '';
+    for (const nameIndex in names) {
+      console.log(names[nameIndex]);
+      namesListElement.appendChild(
+          createListElement('The ' + (+nameIndex + 1).toString() + '-th name is ' + names[nameIndex]));
```

Comment:
http://go/mdn/JavaScript/Reference/Template_literals

You could make the code more readable with template literals. It's supported by all modern browser directly.

Reply:
Thank you, I changed the string.